content/guides/applications/custom-apps/jwt-setup.md

```diff
@@ -41,25 +41,32 @@ Select **Create New App**.
 Select **Custom App** from the list of application types. A modal will appear to
 prompt a selection for the next step.
 
-<ImageFrame border width="400" center>
+<ImageFrame border center>
   ![Auth selection screen](../images/select-app-type.png)
 </ImageFrame>
 
 ### 3. Select the type of authentication and app name
 
-Select **Server Authentication (with JWT)** and provide a unique name for your
-application. Click **Create App**.
+Select **Server Authentication (with JWT)** if you would like to verify 
+application identity with a key pair. Select 
+**Server Authentication (Client Credentials Grant)** if you would like to verify
```

Comment:
Might be worth linking this to /guides/authentication/jwt/without-sdk/#client-credentials-grant for further information.